pkg/didcomm/protocol/issuecredential/models.go

```diff
@@ -20,23 +20,18 @@ type ProposeCredential struct {
 	// CredentialProposal is an optional JSON-LD object that represents
 	// the credential data that the Prover wants to receive.
 	CredentialProposal PreviewCredential `json:"credential_proposal,omitempty"`
-	// SchemaIssuerDid is an optional filter to request credential based on a particular Schema issuer DID.
-	SchemaIssuerDid string `json:"schema_issuer_did,omitempty"`
-	// SchemaID is an optional filter to request credential based on a particular Schema.
-	// This might be helpful when requesting a version 1 passport instead of a version 2 passport, for example.
-	SchemaID string `json:"schema_id,omitempty"`
-	// SchemaName is an optional filter to request credential based on a schema name.
-	// This is useful to allow a more user-friendly experience of requesting a credential by schema name.
-	SchemaName string `json:"schema_name,omitempty"`
-	// SchemaVersion is an optional filter to request credential based on a schema version.
-	// This is useful to allow a more user-friendly experience of requesting a credential by schema name and version.
-	SchemaVersion string `json:"schema_version,omitempty"`
-	// CredDefID is an optional filter to request credential based on a particular Credential Definition.
-	// This might be helpful when requesting a commercial driver's license instead of
-	// an ordinary driver's license, for example.
-	CredDefID string `json:"cred_def_id,omitempty"`
-	// IssuerDid is an optional filter to request a credential issued by the owner of a particular DID.
-	IssuerDid string `json:"issuer_did,omitempty"`
+	// Formats contains an entry for each filter~attach array entry, providing the the value of the attachment @id
+	// and the verifiable credential format and version of the attachment.
+	Formats []Format `json:"formats,omitempty"`
```

Comment:
just curious - don't we have any bdd tests to show usage of these attributes ?

Reply:
No, the usage depends on the specific implementation.